stanford/roles/cluster/defaults/main.yml

```diff
@@ -335,3 +345,5 @@ CLUSTER_ELBS:
             cidr_ip: '0.0.0.0/0'
       subnets:
         - "{{ COMMON_DEPLOYMENT }}-subnet-public-{{ CLUSTER_NUMBER }}"
+
+INSTALL_INSTANCES: false
```

Comment:
We should default to the most common case, which for the time being (and I suspect foreseeable future) is the `ec2` case.
That means we can remove it from all of the above files and then just selectively override it in the applicable cases (`rds`, `emr`).

Not only is this easier/simpler for now, but this will hopefully make future work easier as I assume we'll always have more `ec2` instances than we do `rds` or `emr` clusters.